test/newNameLength.test.ts

```diff
@@ -0,0 +1,67 @@
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '../lib';
+import { kMaxLength } from 'buffer';
```

Comment:
make sure `make lint` passed. Unused imports, etc.

Reply:
I am now being sure to check that